### PR TITLE
[zh-cn] Update deployment.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

**Comment by windsonsea** [comment]
Hi, this file has some more inconsistency with en. Could you please check it through? It's not encouraged to change tiny and part of changes.

The first option is to use `./scripts/lsync.sh`, the next is 'vimdiff'.

If use vimdiff, you may get the similar output:

image

From above comparison, you can find some more differences between en and zh.

**Comment by gaogao101** [comment]
@windsonsea Thanks for your help. The modification has been completed.

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -67,14 +67,14 @@ The following are typical use cases for Deployments:
...
-* [Pause the Deployment](#pausing-and-resuming-a-deployment) to apply multiple fixes to its PodTemplateSpec and then resume it to start a new rollout.
+* [Pause the rollout of a Deployment](#pausing-and-resuming-a-deployment) to apply multiple fixes to its PodTemplateSpec and then resume it to start a new rollout.
 * [Use the status of the Deployment](#deployment-status) as an indicator that a rollout has stuck.
 * [Clean up older ReplicaSets](#clean-up-policy) that you don't need anymore.
 -->
 * 如果 Deployment 的当前状态不稳定，[回滚到较早的 Deployment 版本](#rolling-back-a-deployment)。
   每次回滚都会更新 Deployment 的修订版本。
 * [扩大 Deployment 规模以承担更多负载](#scaling-a-deployment)。
-* [暂停 Deployment ](#pausing-and-resuming-a-deployment) 以应用对 PodTemplateSpec 所作的多项修改，
+* [暂停 Deployment的展示 ](#pausing-and-resuming-a-deployment) 以应用对 PodTemplateSpec 所作的多项修改，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:77]:
```suggestion
* [暂停 Deployment 的上线](#pausing-and-resuming-a-deployment)以应用对 PodTemplateSpec 所作的多项修改，
```
for term consistency in a sentence.

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 2]
```diff
@@ -102,7 +102,6 @@ In this example:
 -->
 * 创建名为 `nginx-deployment`（由 `.metadata.name` 字段标明）的 Deployment。
 * 该 Deployment 创建三个（由 `.spec.replicas` 字段标明）Pod 副本。
-* `.spec.selector` 字段定义了 Deployment 如何查找要管理的 Pod。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:105]: pls keep consistency between en and zh.

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 3]
```diff
@@ -1626,7 +1626,7 @@ Deployment progress has stalled.
 
 <!--
 The following `kubectl` command sets the spec with `progressDeadlineSeconds` to make the controller report
-lack of progress for a Deployment after 10 minutes:
+lack of progress of a rollout for a Deployment after 10 minutes:
 -->
 以下 `kubectl` 命令设置规约中的 `progressDeadlineSeconds`，从而告知控制器
 在 10 分钟后报告 Deployment 没有进展：
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:1632]:
```suggestion
在 10 分钟后报告 Deployment 的上线没有进展：
```

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 4]
```diff
@@ -1683,9 +1683,9 @@ Deployment 不执行任何操作。更高级别的编排器可以利用这一设
...
 {{< note >}}
 <!--
-If you pause a Deployment rollout, Kubernetes does not check progress against your specified deadline. You can
-safely pause a Deployment in the middle of a rollout and resume without triggering the condition for
-exceeding the deadline.
+If you pause a Deployment rollout, Kubernetes does not check progress against your specified deadline. 
+You can safely pause a Deployment rollout in the middle of a rollout and resume without triggering 
+the condition for exceeding the deadline.
 -->
 如果你暂停了某个 Deployment 上线，Kubernetes 不再根据指定的截止时间检查 Deployment 进展。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:1690]:
```suggestion
如果你暂停了某个 Deployment 上线，Kubernetes 不再根据指定的截止时间检查 Deployment 上线的进展。
```

**Review by windsonsea — COMMENTED** [review]
Do not ignore any word when you perform localization.

It's a good choice to keep consistent and sync with en timely.

**Comment by gaogao101** [comment]
@windsonsea The new modification is finished. Thank you very much.

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 5]
```diff
@@ -67,14 +67,14 @@ The following are typical use cases for Deployments:
...
-* [Pause the Deployment](#pausing-and-resuming-a-deployment) to apply multiple fixes to its PodTemplateSpec and then resume it to start a new rollout.
+* [Pause the rollout of a Deployment](#pausing-and-resuming-a-deployment) to apply multiple fixes to its PodTemplateSpec and then resume it to start a new rollout.
 * [Use the status of the Deployment](#deployment-status) as an indicator that a rollout has stuck.
 * [Clean up older ReplicaSets](#clean-up-policy) that you don't need anymore.
 -->
 * 如果 Deployment 的当前状态不稳定，[回滚到较早的 Deployment 版本](#rolling-back-a-deployment)。
   每次回滚都会更新 Deployment 的修订版本。
 * [扩大 Deployment 规模以承担更多负载](#scaling-a-deployment)。
-* [暂停 Deployment ](#pausing-and-resuming-a-deployment) 以应用对 PodTemplateSpec 所作的多项修改，
+* [暂停 Deployment 的上线 ](#pausing-and-resuming-a-deployment) 以应用对 PodTemplateSpec 所作的多项修改，
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:77]:
```suggestion
* [暂停 Deployment 的上线](#pausing-and-resuming-a-deployment)以应用对 PodTemplateSpec 所作的多项修改，
```

**Comment by gaogao101** [comment]
@Sea-n The new modification is finished. Thank you very much.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**